src/main/resources/application.yml

```diff
@@ -30,6 +30,7 @@ micronaut:
   brapi:
     server:
       url: ${brapi.server.url:`https://test-server.brapi.org/`}
+      reference-source: ${brapi.server.reference-source:`breeding-insight.org`}
```

Comment:
Sorry I didn't catch this sooner.  It may be cleaner to have properties that we've defined NOT be under the `micronaut` namespace (see `web` at the bottom of `application.yml` as an example).